src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -338,6 +339,14 @@ def display_shared_memory_warning(warning_message: str) -> None:
     return Constant(_value, shared_memory=_shared_memory)
 
 
+@constant.register
+def _(
+    tensor: Tensor,
+    shared_memory: bool = False,
+) -> Constant:
+    return Constant(tensor, shared_memory=shared_memory)
```

Comment:
To keep this function aligned, shouldn't it be decorated with "nameable_op" as well?
```suggestion
@constant.register
@namable_op
def _(
    tensor: Tensor,
    shared_memory: bool = False,
    name: Optional[str] = None
) -> Constant:
    return Constant(tensor, shared_memory=shared_memory)
```

Reply:
Yes, it should
added this decorator and `name` argument.
Thank you for catching that.